File: backend/components/metiers/threed/utils/helpers/fallback/fallback.js
// fallback.js
// Fallback helper JS pour Threed – exemple clé en main

/**
 * Fallback minimal : retourne une valeur par défaut si la fonction échoue
 * @param {Function} fn
 * @param {*} defaultValue
 * @returns {*}
 */
function fallbackHelper(fn, defaultValue) {
  try {
    return fn();
  } catch {
    return defaultValue;
  }
}

module.exports = { fallbackHelper };

File: backend/components/metiers/threed/utils/helpers/fallback/fallback.py
# fallback.py
# Fallback helper Python pour Threed – exemple clé en main


def fallback_helper(fn, default_value):
    """
    Fallback minimal : retourne une valeur par défaut si la fonction échoue
    """
    try:
        return fn()
    except Exception:
        return default_value
